### PR TITLE
homework number two

### PR DESCRIPTION
**Pull request by kihors**

### CODE REVIEW

File: src/homeWorkSecond/CheckingExpression.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package homeWorkSecond;
+
+public class CheckingExpression {
+
+    public static void main(String[] args) {
+        System.out.println(doubleExpression(2.02, 5.03, 7.05));
+    }
+
+    private static boolean doubleExpression(double firstValue, double secondValue, double sum) {
+        double permissibleError = 0.0001;
```
Comment (Sofasmile) [inline src/homeWorkSecond/CheckingExpression.java]: move permissibleError into class and make as static final variable

File: src/homeWorkSecond/TruthVerification.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package homeWorkSecond;
+
+public class TruthVerification {
+
+    public static void main(String[] args) {
+        System.out.println(booleanExpression(true, false,
+                false, true));
+    }
+
+    private static boolean booleanExpression(boolean firstArgument, boolean secondArgument,
```
Comment (Sofasmile) [inline src/homeWorkSecond/TruthVerification.java]: make another implementation
with boolean operators

File: src/homeWorkSecond/CheckingExpression.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package homeWorkSecond;
```
Comment (Sofasmile) [inline src/homeWorkSecond/CheckingExpression.java]: CS

Comment (kihors) [reply src/homeWorkSecond/CheckingExpression.java]: The name of my package is wrong?
how better? "homework.second" or "womeworkSecond"??

File: src/homeworkSecond/CheckingExpression.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package homeworkSecond;
+
+public class CheckingExpression {
+
+    public static void main(String[] args) {
+        System.out.println(doubleExpression(3.02, 5.03, 8.05));
+    }
+
+    private static final double permissibleError = 0.0001;
```
Comment (Sofasmile) [inline src/homeworkSecond/CheckingExpression.java]: read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

File: src/homeworkSecond/TruthVerification.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package homeworkSecond;
+
+public class TruthVerification {
+
+    public static void main(String[] args) {
+        System.out.println(booleanExpression(true, false,
+                false, true));
+    }
+
+    private static boolean booleanExpression(boolean firstArgument, boolean secondArgument,
```
Comment (Sofasmile) [inline src/homeworkSecond/TruthVerification.java]: you can make it more simply

**Review by Sofasmile — COMMENTED** [review]
resolve comments

File: src/homeworkSecond/CheckingExpression.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package homeworkSecond;
+
+public class CheckingExpression {
+
+    public static void main(String[] args) {
+        System.out.println(doubleExpression(3.02, 5.03, 8.05));
+    }
+
+    private static final double error = 0.0001;
```
Comment (Sofasmile) [inline src/homeworkSecond/CheckingExpression.java]: constant names write with uppercase
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

**Review by Sofasmile — APPROVED** [review]
please, change the name of constant!

File: src/homeworkSecond/CheckingExpression.java (hunk 6 above)
Comment (kihors) [reply src/homeworkSecond/CheckingExpression.java]: Yes of course! Excuse me, please!

**Pull request merged**